test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/util.py

```diff
@@ -23,8 +24,29 @@
 BATCH_CSV = os.path.join('data', 'batch.csv')
 
 
+def _botocore_resolver():
+    """
+    Get the DNS suffix for the given region.
+    :return: endpoint object
+    """
+    loader = botocore.loaders.create_loader()
+    return botocore.regions.EndpointResolver(loader.load_data('endpoints'))
+
+
+def get_ecr_registry(account, region):
+    """
+    Get prefix of ECR image URI
+    :param account: Account ID
+    :param region: region where ECR repo exists
+    :return: AWS ECR registry
+    """
+    endpoint_data = _botocore_resolver().construct_endpoint('ecr', region)
+    return '{}.dkr.{}'.format(account, endpoint_data['hostname'])
```

Comment:
nit: f strings and format used in same file, my vote is to use f strings but if you want to still use .format, let's make it consistent

Reply:
Will change.